### PR TITLE
Adds more money items, up to one million dollars.

### PR DESCRIPTION
**Pull request by VashVandez**
Adds more money items for events that call for amounts of cash that's larger than a thousand.
# About the pull request

Adds additional spacecash items to make it easier for staff to spawn in larger amounts of money for the purposes of RP events.

# Explain why it's good for the game

Who doesn't like money?

### CODE REVIEW

**Comment by DOC-surge** [comment]
i think this is a good idea because it's some of the small things that can be useful like sometimes stacked money can sometimes be a pain and this a can. be a quick fix.

File: code/modules/economy/cash.dm [hunk 1]
```diff
@@ -167,9 +167,39 @@
 /obj/item/spacecash/c1000/counterfeit
 	// If you're going to slap down huge sums of cash in easily reachable spots for RP or flavor reasons, make them counterfeit, so they don't break the black market - they will be rejected.
 	counterfeit = TRUE
+	
+/obj/item/spacecash/c10000
+	name = "10,000 dollars"
+	icon_state = "spacecash1000"
```
Comment (harryob) [inline code/modules/economy/cash.dm:173]: you might just wanna make some sprites for these, given the sprites should be incredibly easy to just,,, make bigger

**Review by sleepynecrons — CHANGES_REQUESTED** [review]
move the 10k and 100k piles closer to the center so they're not hugging the sprite border 
image
stack up the 1 mil pile closer to the example on the far right so that it isn't also touching the top and bottom borders

**Comment by VashVandez** [comment]
> move the 10k and 100k piles closer to the center so they're not hugging the top and bottom sprite border image stack up the 1 mil pile closer to the example on the far right so that there's more of a progression and it isn't also touching the top and bottom borders

jb3vHTY
N6nhF0a
xq50dx4

**Review by sleepynecrons — APPROVED** [review]
the 10k and 100k shapes themselves were fine but this works too. sprites approved

File: code/modules/economy/cash.dm [hunk 2]
```diff
@@ -165,11 +164,38 @@
...
+	worth = 1000000
+	
+/obj/item/spacecash/c1000000/counterfeit
 	// If you're going to slap down huge sums of cash in easily reachable spots for RP or flavor reasons, make them counterfeit, so they don't break the black market - they will be rejected.
 	counterfeit = TRUE
 
 /proc/spawn_money(sum, spawnloc, mob/living/carbon/human/human_user as mob)
-	if(sum in list(1000,500,200,100,50,20,10,1))
+	if(sum in list(1000000,100000,10000,1000,500,200,100,50,20,10,1))
 		var/cash_type = text2path("/obj/item/spacecash/c[sum]")
```
Comment (Zonespace27) [inline code/modules/economy/cash.dm:199]: this isn't your fault but i've only looked at two lines of this proc and i hate it already

**Pull request closed without merging**